src/components/component-playground.jsx

```diff
@@ -173,6 +177,10 @@ module.exports = React.createClass({
     </ul>
   },
 
+  onSplitPaneChange: function(size) {
```

Comment:
Public methods should be placed in between `_renderMethods` and `_privateMethods`